e2e/support/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -110,18 +110,35 @@ export function openTable({
   );
 }
 
+/**
+ *
+ * @typedef {{mode?: "notebook", limit?: number, callback?: function }} OpenTablesProps
```

Comment:
We should probably debounce the row limit input because typing N digits in it will cause N requests to be dispatched and will noticeably slow down the UI while the requests are being processed. Try pressing and holding a digit key in it.

I found it when I was trying to type "123" and then noticed the UI gets laggy.

https://github.com/metabase/metabase/assets/6830683/cbc0c141-cdd3-4dd1-adce-14d27e442be9

Reply:
I think it should update the query on blur

Reply:
I think it should be fine to fix this step in a follow-up PR

Reply:
https://github.com/metabase/metabase/pull/40630